corehq/apps/users/views/mobile/users.py

```diff
@@ -560,7 +563,7 @@ def can_access_all_locations(self):
 
     @property
     def can_bulk_edit_users(self):
-        return has_privilege(self.request, privileges.BULK_USER_MANAGEMENT)
+        return has_privilege(self.request, privileges.BULK_USER_MANAGEMENT) and not self.request.is_view_only
```

Comment:
This controls both the download and upload buttons: https://github.com/dimagi/commcare-hq/blob/f895fea74f6cc9914dbf30275c075bef58e5840d/corehq/apps/users/templates/users/mobile_workers.html#L67-L74 Should it be limited to controlling the upload?

Reply:
done on purpose